.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 #          - version: '1' # SymbolicWedderburn fails
 #            os: ubuntu-latest
 #            arch: x86
-          - version: '1.6'
+          - version: '1.10'
```

Comment:
Is it intended to not test on v1.6 anymore but still allow it in the Project file?